src/spikeinterface/preprocessing/whiten.py

```diff
@@ -40,6 +41,12 @@ class WhitenRecording(BasePreprocessor):
     M : 1d np.array or None, default: None
         Pre-computed means.
         M can be None when previously computed with apply_mean=False
+    regularize: bool, default False
+        Boolean to decide if we want to regularize the covariance matrix, using the GraphicalLassoCV() method
+        of sklearn
+    regularize_kwargs: None or dict
+        Dictionary of the parameters that could be provided to the GraphicalLassoCV() method of sklearn, if
+        the covariance matrix needs to be regularized
```

Comment:
```suggestion
    regularize : bool, default: False
        Boolean to decide if we want to regularize the covariance matrix, using the GraphicalLassoCV() method
        of sklearn
    regularize_kwargs : None or dict
        Dictionary of the parameters that could be provided to the GraphicalLassoCV() method of sklearn, if
        the covariance matrix needs to be regularized
```

Let's stick to the correct docstring formatting! Otherwise @chrishalcrow will be very mad!